src/Core/Settings.h

```diff
@@ -773,6 +773,8 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     \
     M(UInt64, input_format_allow_errors_num, 0, "Maximum absolute amount of errors while reading text formats (like CSV, TSV). In case of error, if at least absolute or relative amount of errors is lower than corresponding value, will skip until next line and continue.", 0) \
     M(Float, input_format_allow_errors_ratio, 0, "Maximum relative amount of errors while reading text formats (like CSV, TSV). In case of error, if at least absolute or relative amount of errors is lower than corresponding value, will skip until next line and continue.", 0) \
+    M(String, input_format_record_errors_file_path, "_input_format_error_rows_", "Path of the file used to record errors while reading text formats (CSV, TSV).", 0) \
+    M(ErrorsOutputFormat, errors_output_format, "CSV", "Method to write Errors to text output. Possible values: 'CSV'.", 0) \
```

Comment:
Let's make it just String, we can support arbitrary format that is supported for output in ClickHouse, see comment in InputFormatErrorsLogger

Reply:
But how do we handle the wrong format?

Reply:
We can check if such format is supported (`FormatFactory` has method `isOutputFormat`) and if format is not supported, we can do one of the following:
1) Throw an exception, but it's not the best way
2) Skip recording errors and write message in log/stderr
3) Use default format, for example CSV
I think 3-rd variant wil be the best

Reply:
Yes, I agree with you.